dev/sparktestsupport/modules.py

```diff
@@ -778,6 +778,68 @@ def __hash__(self):
         # ml unittests
         "pyspark.ml.tests.connect.test_connect_function",
         "pyspark.ml.tests.connect.test_parity_torch_distributor",
+        # pandas-on-Spark unittests
```

Comment:
`pyspark-connect` takes 2~3 hours after adding the PS related unittests, I guess we can add new modules `pyspark-connect-pandas` and `pyspark-connect-pandas-slow` @itholic

Reply:
Let me address within follow-up PR. Thanks!